torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11418,6 +11418,7 @@ def ref_pairwise_distance(input1, input2):
            supports_autograd=True,
            sample_inputs_func=sample_inputs_glu,
            dtypes=floating_types(),
+           dtypesIfCPU=floating_types_and(torch.bfloat16),
```

Comment:
'dtypesIfCPU' keyword has been removed, just modify 'dtypes' will clear the CI failure.

`dtypes=floating_types_and(torch.bfloat16),`

Reply:
Thanks @mingfeima, I will fix and rebase it.